### ISSUE TITLE
Reading Tips in Visual Studio Express 2010

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#943]
**Reported by parham on 2010-09-25 13:12**
When pressing ctrl+I in the Visual Studio Express 2010 editor, a tip will appear on the screen. however, I am unable to find this tip with NVDA. Also, I can not read other kind of tips, like the one that appears by pressing ctrl+shift+space and lists the arguments of a function.

**Comment by leonardder** [comment]
@mohammad-suliman says in #6043 that this needs an external fix by MS to work properly. This is very unlikely to happen, since we now have VS 2017.

**Comment by jcsteh** [comment]
Closing, since 2010 is very old and this is unlikely to ever get fixed.
